app/src/organisms/Devices/ProtocolRun/SetupModules/SetupModulesList.tsx

```diff
@@ -251,35 +265,44 @@ export function ModulesListItem({
     )
   }
 
-  const RenderModuleStatus = (): JSX.Element => {
-    const handleCalibrate = (): void => {
-      setShowModuleWizard(true)
-    }
-    if (attachedModuleMatch == null) {
-      return (
-        <StatusLabel
-          status={moduleConnectionStatus}
-          backgroundColor={COLORS.warningBackgroundLight}
-          iconColor={COLORS.warningEnabled}
-          textColor={COLORS.warningText}
-        />
-      )
-    } else if (attachedModuleMatch.moduleOffset?.last_modified != null) {
-      return (
-        <StatusLabel
-          status={moduleConnectionStatus}
-          backgroundColor={COLORS.successBackgroundLight}
-          iconColor={COLORS.successEnabled}
-          textColor={COLORS.successText}
-        />
-      )
-    } else {
-      return (
-        <TertiaryButton onClick={handleCalibrate}>
+  let renderModuleStatus: JSX.Element
```

Comment:
it is a bit easier to read in my opinion to initialize the `renderModuleStatus` to = the else, then you can delete the else.

```suggestion
  let renderModuleStatus: JSX.Element = (
      <>
        <TertiaryButton
          {...targetProps}
          onClick={() => setShowModuleWizard(true)}
          disabled={!calibrationStatus?.complete}
        >
          {t('calibrate_now')}
        </TertiaryButton>
        {!calibrationStatus?.complete && calibrationStatus?.reason != null ? (
          <Tooltip tooltipProps={tooltipProps}>
            {calibrationStatus.reason === 'attach_pipette_failure_reason'
              ? t('attach_pipette_before_module_calibration')
              : t('calibrate_pipette_before_module_calibration')}
          </Tooltip>
        ) : null}
      </>
    )
```